client/components/AggregatedReviews.jsx

```diff
@@ -1,9 +1,75 @@
 import React from 'react';
-import style from './styles/AggregatedStyle.css';
+import $ from 'jquery';
+import './styles/AggregatedStyle.css';
+import emptyStar from './styles/icons/star-clear.svg';
+import halfStar from './styles/icons/star-half.svg';
+import fullStar from './styles/icons/star.svg';
 
 const AggregatedReviews = (props) => {
+  const renderStars = function (rating, outerDiv) {
+
+    // don't add stars on re-render if they already exist
+    if ($(outerDiv).children().length !== 0) {
+      return;
+    }
+    // numStars is used to ensure that 5 stars are always rendered
+    let numStars = 0;
+    // this loop renders all initial full stars
+    while (rating / 1 >= 1) {
+      $(outerDiv).append(`<img class="star" src=${fullStar} />`);
```

Comment:
Maybe this line could be something like:`outputArray.push(<img className="star" src={fullStar} />)`? Hopefully I'm not missing anything with this approach.